include/envoy/stream_info/stream_info.h

```diff
@@ -494,6 +499,17 @@ class StreamInfo {
    *         failed.
    */
   virtual const std::string& upstreamTransportFailureReason() const PURE;
+
+  /**
+   * @param request headers.
```

Comment:
nit: It's hit-or-miss in this file, but these should have the parameter name (e.g. `@param headers request headers.`). There's a similar pattern for return types as well.

Reply:
done